src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,84 @@
 import React from 'react';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+import PropTypes from 'prop-types';
+
+export const NewCommentForm = ({
+  name,
+  setName,
+  email,
+  setEmail,
+  comment,
+  setComment,
+  addToList,
+}) => {
+  const formSubmit = (event) => {
+    event.preventDefault();
+    addToList();
+    setName('');
+    setEmail('');
+    setComment('');
+  };
+
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={event => formSubmit(event)}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          value={name}
+          onChange={event => setName(event.target.value)}
```

Comment:
```suggestion
          onChange={({ target }) => setFormData(prevData => ({
             ...prevData,
             [target.name]: target.value,
          }))}
```
It's even better solution. this way you could move this to the method, and pass single function in every input, instead of creating anonymous one for each input